zeek-ocsf/package.yaml

```
@@ -485,22 +485,24 @@ pipelines:
         protocol_name: "udp",
         protocol_num: 17,
       }
-      if zeek.client_addr.is_v6() or zeek.client_addr.is_v6() {
+      if not zeek.id?.dest_h? == null {
```

```suggestion
      if not zeek.id?.resp_h? == null {
```

I've never seen `dest_h`. Can you add a comment what you are trying to do?